models/model.php

```diff
@@ -2058,12 +2045,14 @@ function event_calendar_queue_reminders() {
 			}
 		}
 	}
+
+	elgg_set_ignore_access($ia);
 }
 
 function event_calendar_repeat_reminder_log($e, $start) {
 	// this simple log just uses annotations on the event
 	// TODO - remove log entries for past events
-	create_annotation($e->guid, 'repeat_reminder_log_item', $start, '', 0, ACCESS_PUBLIC);
+	create_annotation($e->guid, 'repeat_reminder_log_item', $start, '', 0, ACCESS_PRIVATE);
```

Comment:
These annotations are only for internal use, so I don't think public access is necessary. Creating them with private access  seems to me to fit better.